Gem/Fiat/Views/BuyAssetInputField.swift

```diff
@@ -0,0 +1,63 @@
+// Copyright (c). Gem Wallet. All rights reserved.
+
+import SwiftUI
+import Style
+
+struct BuyAssetInputField: View {
+    @Binding private var value: Double
+
+    enum FiatField: Int, Hashable, Identifiable {
+        case fiat
+        var id: String { String(rawValue) }
+    }
+
+    private var focusedFieldBinding: FocusState<FiatField?>.Binding
+
+    private var textBinding: Binding<String> {
+        Binding<String>(
+            get: {
+                String(format: "%.0f", value)
```

Comment:
What happens if you type $10.50, will it format properly?

Reply:
The %.0f format specifier rounds the Double value to the nearest integer and then formats it as a string without any decimal places. Since the value is 10.5, it gets rounded down to 10 when converted to a string using this format.